mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -1,31 +1,49 @@
 """This module defines the outbound transmission component."""
 
-import logging
 from pathlib import Path
+from ssl import SSLError
+from typing import Dict
+
 from comms import transmission_adaptor
+from comms.common_https import CommonHttps
+from tornado import httpclient
+from utilities import integration_adaptors_logger as log
+
+logger = log.IntegrationAdaptorsLogger("OUTBOUND_TRANSMISSION")
 
-import requests
+
+class MaxRetriesExceeded(Exception):
+    pass
 
 
 class OutboundTransmission(transmission_adaptor.TransmissionAdaptor):
+    errors_not_to_retry = [
+        httpclient.HTTPClientError,
+        SSLError
+    ]
+
     """A component that sends HTTP requests to a remote MHS."""
 
-    def __init__(self, certs_dir):
+    def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs: str, max_retries: int):
         """Create a new OutboundTransmission that loads certificates from the specified directory.
         :param certs_dir: A string containing the path to the directory to load certificates from.
+        :param client_cert: A string containing the name of the client certificate file in the certs directory.
+        :param client_key: A string containing the name of the client private key file in the certs directory.
+        :param ca_certs: A string containing the name of the certificate authority certificate file in the certs directory.
+        :param max_retries: An integer with the value of the max number times to retry sending the request.
         """
-        client_cert_path = str(Path(certs_dir) / "client.cert")
-        client_key_path = str(Path(certs_dir) / "client.key")
-        self._cert = (client_cert_path, client_key_path)
-        self._verify = str(Path(certs_dir) / "client.pem")
+        self._client_cert = str(Path(certs_dir) / client_cert)
+        self._client_key = str(Path(certs_dir) / client_key)
+        self._ca_certs = str(Path(certs_dir) / ca_certs)
+        self._max_retries = max_retries
 
-    def make_request(self, interaction_details, message):
+    async def make_request(self, interaction_details: Dict[str, str], message: str) -> httpclient.HTTPResponse:
```

Comment:
In combination with other comments, this should end up being changed to something like:
```suggestion
    async def make_request(self, url: str, message: str) -> httpclient.HTTPResponse:
```